patches/tModLoader/Terraria/ModLoader/RecipeLoader.cs

```diff
@@ -57,6 +57,28 @@ public static class RecipeLoader
 			}
 		}
 
+		/// <summary>
+		/// Deletes the recipes flagged for deletion
+		/// </summary>
+		internal static void DeleteRecipes() {
```

Comment:
I'm not a fan of the idea of recipes being deleted.
This easily can run away in to some weird cross mod interactions with null/missing objects.

Let's focus on disabling recipes instead by adding an always false condition to them. This poses less risk to cause unintended cross-mod interactions